tests/main/snap-seccomp-syscalls/task.yaml

```diff
@@ -18,6 +18,9 @@ restore: |
     rm -f syscalls.*
 
 execute: |
+    # XXX: this should be part of our images
+    apt install gperf -y
```

Comment:
Should we add a "apt remove" in the restore just for cleanliness?

Reply:
I think we can leave it behind for now. It's pretty harmless

Reply:
Mild preference for being tidy.